Week2/homework/map_filter.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
 const numbers = [1, 2, 3, 4];
+/*
+const newNumbers = [];
+
+for (let i = 0; i < numbers.length; i++) {
+  if (numbers[i] % 2 !== 0) {
+    newNumbers.push(numbers[i] * 2);
+  }
+}
+console.log('The doubled numbers are', newNumbers);
+*/
+function isOdd(num) {
+  return num % 2 !== 0;
+}
+const filteredNumber = (numbers.filter(isOdd));
+function double(num) {
+  return num * 2;
+}
+
+console.log('The doubled numbers are', filteredNumber.map(double));
```

Comment:
This is a lot of code for something really simple. I would just use anonymous arrow functions and method chaining, e.g.:

```js
const newNumbers = numbers
  .filter(number => number % 2 !== 0)
  .map(num => num * 2);
```